js/app/product/form.scss

```diff
@@ -0,0 +1,8 @@
+.sticky-buttons{
```

Comment:
You should actually move this class into `js/widgets/admin.scss`, this way it will be available throughout the whole admin interface.